### PR TITLE
dashboard: prefer alias for account transaction/balance link filters

### PR DESCRIPTION
**Pull request by dominic**

### CODE REVIEW

**Comment by dominic** [comment]
@jeffomatic PTAL

File: dashboard/src/features/accounts/components/Show.jsx [hunk 1]
```diff
@@ -90,14 +89,24 @@ const mapStateToProps = (state, ownProps) => ({
...
-  showTransactions: (id) => {
+  showTransactions: (item) => {
+    let filter = `inputs(account_id='${item.id}') OR outputs(account_id='${item.id}')`
+    if (item.alias) {
+      filter = `inputs(account_alias='${item.alias}') OR outputs(account_alias='${item.alias}')`
+    }
+
     dispatch(actions.transaction.pushList({
-      filter: `inputs(account_id='${id}') OR outputs(account_id='${id}')`
+      filter: filter
```
Comment (jeffomatic) [inline dashboard/src/features/accounts/components/Show.jsx]: Can just use `{filter}`

File: dashboard/src/features/accounts/components/Show.jsx [hunk 2]
```diff
@@ -90,14 +89,24 @@ const mapStateToProps = (state, ownProps) => ({
...
-  showBalances: (id) => {
+  showBalances: (item) => {
+    let filter = `account_id='${item.id}'`
+    if (item.alias) {
+      filter = `account_alias='${item.alias}'`
+    }
+
     dispatch(actions.balance.pushList({
-      filter: `account_id='${id}'`
+      filter: filter
```
Comment (jeffomatic) [inline dashboard/src/features/accounts/components/Show.jsx]: Ditto

File: dashboard/src/features/assets/components/Show.jsx [hunk 3]
```diff
@@ -69,12 +69,12 @@ const mapStateToProps = (state, ownProps) => ({
...
   showCirculation: (item) => {
-    let query = `asset_id='${item.id}'`
+    let filter = `asset_id='${item.id}'`
     if (item.alias) {
-      query = `asset_alias='${item.alias}'`
+      filter = `asset_alias='${item.alias}'`
     }
 
-    dispatch(actions.balance.pushList({filter: query}))
+    dispatch(actions.balance.pushList({filter: filter}))
```
Comment (jeffomatic) [inline dashboard/src/features/assets/components/Show.jsx]: Ditto

**Pull request merged**